test/quantization/core/experimental/test_quantizer.py

```diff
@@ -72,14 +72,63 @@ def test_quantize_APoT_k2(self):
         # 0.0215 in tensor2quantize nearest 0.0208 in quantization_levels -> 3 in level_indices
         expected_qtensor = torch.tensor([3, 8, 13, 12], dtype=torch.uint8)
 
-        print(qtensor_data)
-        print(expected_qtensor)
-
         self.assertTrue(torch.equal(qtensor_data, expected_qtensor))
 
-    def test_dequantize(self):
-        with self.assertRaises(NotImplementedError):
-            APoTQuantizer.dequantize(self)
+    r""" Tests dequantize_apot result on random 1-dim tensor
+        and hardcoded values for b, k.
+        Dequant -> quant an input tensor and verify that
+        result is equivalent to input
+        * tensor2quantize: Tensor
+        * b: 4
+        * k: 2
+    """
+    def test_dequantize_quantize_rand_b4(self):
+        # generate random size of tensor2dequantize between 1->20
+        size = random.randint(1, 20)
+
+        # initialize quantize APoT tensor to dequantize:
+        # generate tensor with random values between 0 -> 2**4 = 16
+        # because there are 2**b = 2**4 quantization levels total
+        tensor2dequantize = 16 * torch.rand(size)
+        quantizer = APoTQuantizer(4, 2, 1.0, False)
+        quantizer.data = tensor2dequantize.int()
```

Comment:
why `data` is stored in quantizer? I feel we should store it in a quantized tensor